### PR TITLE
speed up package installation

### PR DESCRIPTION
**Pull request by keachi**
Use
```
package:
  name: '{{ ROLE_packages }}'
```
instead of
```
package:
  name: '{{ item }}'
with_items: '{{ ROLE_packages }}'
```
because this will speed up the package installation.

### CODE REVIEW

File: ansible/tasks/installation.yml [hunk 1]
```diff
@@ -2,6 +2,5 @@
 
 - name: install ansible helper packages
   package:
-    name: '{{ item }}'
+    name: '{{ ansible_packages }}'
```
Comment (karras) [inline ansible/tasks/installation.yml:5]: Just noticed that we should maybe rename that role to not interfere with the `ansible_*` variable namespace.

File: postfix/tasks/installation.yml [hunk 2]
```diff
@@ -2,30 +2,27 @@
 
 - name: remove conflicting packages
   apt:
-    name: '{{ item }}'
+    name: '{{ postfix_packages_removed }}'
```
Comment (karras) [inline postfix/tasks/installation.yml:5]: Variable name should be aligned, either use `removed` or `remove`.

File: postfix/tasks/installation.yml [hunk 3]
```diff
@@ -2,30 +2,27 @@
...
   tags:
     - 'role::postfix'
     - 'role::postfix:install'
-  with_items: '{{ postfix_packages_removed }}'
   when: ansible_os_family == 'Debian'
 
 - name: remove conflicting packages
   package:
-    name: '{{ item }}'
+    name: '{{ postfix_packages_removed }}'
```
Comment (karras) [inline postfix/tasks/installation.yml:15]: Variable name should be aligned, either use `removed` or `remove`.

**Review by karras — CHANGES_REQUESTED** [review]
Just some small adjustments...

**Comment by karras** [comment]
Very nice by the way! Do you think we should create an upstream PR to update the documentation?

File: postfix/tasks/installation.yml (hunk 2 above)
Comment (keachi) [reply postfix/tasks/installation.yml:5]: https://github.com/adfinis-sygroup/ansible-roles/pull/23

File: postfix/tasks/installation.yml (hunk 3 above)
Comment (keachi) [reply postfix/tasks/installation.yml:15]: https://github.com/adfinis-sygroup/ansible-roles/pull/23

File: ansible/tasks/installation.yml (hunk 1 above)
Comment (keachi) [reply ansible/tasks/installation.yml:5]: But this would be another PR.

**Review by karras — APPROVED** [review]

**Comment by karras** [comment]
Can you resolve the conflicts?

**Pull request merged**